core/decorations.py

```diff
@@ -0,0 +1,79 @@
+from functools import reduce
+from LSP.plugin.core.css import css
+from LSP.plugin.core.protocol import Range
+from LSP.plugin.core.sessions import Session
+from LSP.plugin.core.typing import Any, List, Dict, Optional
+from LSP.plugin.core.views import range_to_region, FORMAT_MARKED_STRING, FORMAT_MARKUP_CONTENT, minihtml
+import mdpopups
+import sublime
+
+
+def publishDecorations(session: Session, decorationsParams: Any) -> None:
```

Comment:
Naming convention not quite python-like.

I see that you are doing that in various files. Up to you but I wouldn't try to match the request/notification name.